src/Calculator/App.xaml

```diff
@@ -85,6 +85,22 @@
                     <SolidColorBrush x:Key="EquationBrush13" Color="#FF8E562E"/>
                     <SolidColorBrush x:Key="EquationBrush14" Color="#FF000000"/>
 
+                    <SolidColorBrush x:Key="GraphSettingsErrorBackgroundBrush" Color="#33EB5757"/>
+                    <SolidColorBrush x:Key="GraphSettingsErrorBorderBrush" Color="#FFEB5757"/>
+                    <SolidColorBrush x:Key="TextControlBackgroundFocused" Color="Transparent"/>
```

Comment:
These brushes override system brushes and will modify all textboxes in the app. Their scopes were limited when they were in `EquationInputArea` but not here.

Reply:
Good catch!